settings/common.py

```diff
@@ -87,7 +87,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [APPS_DIR],
```

Comment:
Isn't there any other way to add a `<li>` element to Django admin. I used `django-admin-tools` which was pretty configurable and quick links can be very easily added without updating the html. @deshraj : your thoughts here.

Reply:
@taranjeet do you think that it is a good idea to have Django admin tools for EvalAI? IMO: it would be really easy for the admins to manage different challenges via Admin interface. Let me know what you think about this.

Reply:
I think there should be a admin dashboard for challenge host where in they have some quick actions and settings. Django admin is good but it is only suitable for evalai admins. For challenge host they should be provided with some ui friendly dashboard.